src/components/LogoutLink.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'react-router/lib/Link';
+
+/**
+ * Logout link component
+ * @module LogoutLink
+ */
+class LogoutLink extends React.Component {
+	render() {
+		const {
+			to,
+			children,
+			...other
+		} = this.props;
+
+		return (
+			<Link
+				to={{ pathname: to, query: {logout: true} }}
```

Comment:
i could imagine product-wise wanting a logout link that redirects you to a completely different page or a specific flow. how about making the `to` prop optional and the default being the current URL?